java/build.gradle

```diff
@@ -27,11 +27,13 @@ dependencies {
     implementation 'org.apache.orc:orc-core:1.8.1'
     implementation 'com.github.davidmoten:hilbert-curve:0.2.3'
     implementation 'com.carrotsearch:hppc:0.10.0'
+    implementation "io.github.earcut4j:earcut4j:2.2.2"
```

Comment:
We already depend on JTS, which has a few built-in polygon triangulation utilities: https://github.com/locationtech/jts/tree/master/modules/core/src/main/java/org/locationtech/jts/triangulate/polygon - and I think `PolygonEarClipper` is similar to earcut.  Do you have any quality/performance comparison of those vs. earcut4j? If earcut4j performs better then adding a new dependency seems fine, but if not I'd default to the dependencies we already have.